addons/web_editor/static/src/js/wysiwyg/widgets/chatgpt_prompt_dialog.js

```diff
@@ -32,14 +32,32 @@ export class ChatGPTPromptDialog extends ChatGPTDialog {
                 content: 'What do you need ?',
             }],
             messages: [],
+            updateScroll: false,
         });
         this.promptInputRef = useRef('promptInput');
+        this.modalRef = useChildRef();
         useAutofocus({ refName: 'promptInput' });
         useEffect(() => {
             // Resize the textarea to fit its content.
             this.promptInputRef.el.style.height = 0;
             this.promptInputRef.el.style.height = this.promptInputRef.el.scrollHeight + 'px';
         }, () => [this.state.prompt]);
+        useEffect(() => {
+            // Scroll to the latest message whenever new message
+            // is inserted.
+            const modalEl = this.modalRef.el.querySelector("main.modal-body");
+            const lastMessageEl = modalEl.lastElementChild;
+            let scrollDiff;
+            if (lastMessageEl.clientHeight > modalEl.clientHeight) {
+                scrollDiff = modalEl.scrollHeight - lastMessageEl.clientHeight;
+            } else {
+                scrollDiff = modalEl.scrollHeight - modalEl.clientHeight;
+            }
+            modalEl.scrollTo({
+                top: scrollDiff,
+                behavior: "smooth",
+            })
+        }, () => [this.state.updateScroll]);
```

Comment:
Why need `updateScroll` can't you directly use `this.state.conversationHistory.length`?
```suggestion
        }, () => [this.state.conversationHistory.length]);
```